src/modules/pf3dTracker/src/CMakeLists.txt

```diff
@@ -2,14 +2,17 @@
 # Authors: Lorenzo Natale, Matteo Taiana, Ugo Pattacini
 # CopyPolicy: Released under the terms of the GNU GPL v2.0.
 
+cmake_minimum_required(VERSION 2.6)
+set(PROJECTNAME pf3dTracker)
```

Comment:
This is not necessary. 
If you define `project(pf3dTracker)`, then you have a variable defined `PROJECT_NAME` in your cmake project.